synapse/tests/test_model_inet.py

```diff
@@ -393,6 +393,22 @@ def test_model_inet_postref(self):
             self.eq(pr0[1].get('inet:web:postref:xref:strval'), fiden)
             self.eq(pr0[1].get('inet:web:postref:xref:intval'), None)
 
+    def test_model_inet_postref_postmissingprops(self):
```

Comment:
I updated this test to show that the `inet:web:post` will not be formed, but if someone forms it with the same data, the guids will (obviously) be the same.